src/OSPSuite.Core/Domain/ObjectPathKeywords.cs

```diff
@@ -55,6 +55,12 @@ public static class ObjectPathKeywords
       /// </summary>
       public static readonly string TRANSPORT = addKeyword("TRANSPORT");
 
+      /// <summary>
+      ///    String representing a reference to all floating molecules. The entry will be duplicated.
+      ///    This is typically used in event assignment to change all floating molecules at once
+      /// </summary>
+      public static readonly string ALL_FLOATING = addKeyword("ALL_FLOATING");
```

Comment:
should we name the keyword ALL_FLOATING_MOLECULES and not just ALL_FLOATING?

Reply:
Sure.